SmartDeviceLink/SDLVideoEncoder.m

```diff
@@ -119,7 +125,16 @@ - (void)stop {
 }
 
 - (BOOL)encodeFrame:(CVImageBufferRef)imageBuffer {
-    OSStatus status = VTCompressionSessionEncodeFrame(_compressionSession, imageBuffer, CMTimeMake(self.currentFrameNumber++, 30), kCMTimeInvalid, NULL, (__bridge void *)self, NULL);
+    return [self encodeFrame:imageBuffer pts:kCMTimeInvalid];
+}
+
+- (BOOL)encodeFrame:(CVImageBufferRef)imageBuffer pts:(CMTime)pts {
+    if (!CMTIME_IS_VALID(pts)) {
+        pts = CMTimeMake(self.currentFrameNumber, 30);
```

Comment:
Actually this is from current implementation, I kept it as it is. (Please refer to: https://github.com/smartdevicelink/sdl_ios/blob/release/5.0.0/SmartDeviceLink/SDLVideoEncoder.m#L122)

My thought is that app should provide timestamp along with video frames, especially when RTP format is used.
If it cannot provide, then SDL Proxy may create timestamps by capturing current time when `-sendVideoData:` is called. I was hesitated to add this implementation because it may change behavior even when we use byte-stream format.

I’m aware of `kVTCompressionPropertyKey_ExpectedFrameRate` key in the configuration. But I thought we cannot guarantee that all navi apps add it in the configuration dictionary.

Reply:
Can you add this as an issue? I agree with keeping it as is for now.

Reply:
OK, I created issue https://github.com/smartdevicelink/sdl_ios/issues/717.